tests/integ/test_packaging.py

```diff
@@ -524,6 +563,31 @@ def check_if_package_works() -> str:
         )
 
 
+@pytest.mark.skipif(
+    IS_IN_STORED_PROC,
+    reason="Subprocess calls are not allowed within stored procedures",
+)
+def test_add_packages_unsupported_during_sproc_registration(session):
+    """
+    Assert that unsupported packages can directly be added while registering UDFs.
+    """
+    session.custom_packages_upload_enabled = True
+    with patch.object(session, "_is_anaconda_terms_acknowledged", lambda: True):
+        packages = ["scikit-fuzzy==0.4.2", "snowflake-snowpark-python"]
+        sproc_name = Utils.random_name_for_temp_object(TempObjectType.FUNCTION)
+
+        @sproc(name=sproc_name, packages=packages, return_type=StringType())
+        def check_if_package_works(session_):
```

Comment:
these additions are out of scope for this PR

Reply:
This test is for ensuring that custom_packages_upload_enabled works for sproc registration; will add it in a later PR